frontend/src/app/page.tsx

```diff
@@ -1,113 +1,126 @@
-            rel="noopener noreferrer"
-          >
-            By{' '}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className="dark:invert"
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
+    // 🔴 Background Gradient (Red Tone)
+    <Box
+      sx={{
+        minHeight: "100vh",
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        background: "linear-gradient(to bottom, #E06A67 0%, #E8615D 35%, #E14743 75%)", // Matches soft red gradient
+      }}
+    >
+      {/* 📝 White Login Card with Rounded Corners */}
+      <Paper
+        elevation={10}
+        sx={{
+          p: 4,
+          borderRadius: 5, // Rounded corners
+          width: 380,
+          bgcolor: "#fdf7ed", // Soft cream background
+          textAlign: "center",
+        }}
+      >
+        {/* 🔺 Logo (Big Red Hacks) */}
+        <Box sx={{ display: "flex", justifyContent: "center", mb: 2 }}>
+          <Image src="/BRHLogo.png" alt="Big Red Hacks" width={140} height={60} />
+        </Box>
 
-      <div className="relative flex place-items-center before:absolute before:h-[300px] before:w-[480px] before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[''] after:absolute after:-z-20 after:h-[180px] after:w-[240px] after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[''] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700 before:dark:opacity-10 after:dark:from-sky-900 after:dark:via-[#0141ff] after:dark:opacity-40 before:lg:h-[360px] z-[-1]">
-        <Image
-          className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert"
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
-        />
-      </div>
+        {/* 📄 Login Form */}
+        <Box component="form" sx={{ mt: 2 }}>
+          {/* 📧 Email Input Field */}
+          <TextField
+           fullWidth
+           label="Email"
+           variant="outlined"
+           margin="normal"
+           required
+           sx={{
+             bgcolor: "#fdf7ed", // ✅ Matches the login box's background color
+             borderRadius: 2,
+             "& .MuiOutlinedInput-root": { 
+               borderRadius: 3, // ✅ Rounds the input field like the design
+               "& fieldset": { borderColor: "#d6b8a9" }, // ✅ Light border similar to the design
+               "&:hover fieldset": { borderColor: "#c9a89a" }, // ✅ Darker border on hover
+               "&.Mui-focused fieldset": { borderColor: "#b88c7a" }, // ✅ Active input border
+             },
+           }}
+          />
+          
+          {/* 🔒 Password Input Field */}
+          <TextField
```

Comment:
The password field shouldn't display the actual characters when someone types in it. It should display dots for each character like a traditional password field.